README.md

```diff
@@ -182,6 +183,18 @@ It is possible to use a custom [`ServiceAccount`](https://kubernetes.io/docs/ref
 
 **Important**: the Operator handles the creation of default resources necessary to run. If you choose to use a custom ServiceAccount be sure to also configure [`Role`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole) and [`RoleBinding`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding) resources.
 
+## Control random admin password generation
```

Comment:
```suggestion
## Control Random Admin Password Generation
```
We should capitalize this one in order to maintain consistency with the other headers.